trestle/common/model_utils.py

```diff
@@ -595,3 +596,14 @@ def dict_to_parameter(param_dict: Dict[str, Any]) -> common.Parameter:
                     if value not in choices:
                         raise TrestleError(f'Value provided for parameter, {value} is not in choices: {choices}.')
         return common.Parameter(**param_dict)
+
+    @staticmethod
+    def update_timestamp(model: TopLevelOscalModel) -> None:
+        """Update the LastModified timestamp in top level model to now."""
+        model.metadata.last_modified.__root__ = datetime.now().astimezone()
+
+    @staticmethod
+    def model_age(model: TopLevelOscalModel) -> int:
+        """Find time in seconds since LastModified timestamp."""
+        dt = datetime.now().astimezone() - model.metadata.last_modified.__root__
```

Comment:
Is it worth checking for the `None` here since `last_modified` is optional?

Reply:
good catch - fixed